grafana/templates/custom.js

```diff
@@ -92,7 +92,7 @@ Custom.prototype._processOptions = function _processOptions() {
         throw new SyntaxError("default value not found in options list")
       }
       this.state.current = defaultOption
-    } else if (!this.state.current && !this.state.includeAll) {
+    } else if (!this.state.current) {
```

Comment:
I'm pretty sure this will break `includeAll`. For example if `includeAll == true` then it will get ignored and the dash will default to the `newOptions[0]`, right?

i remember playing with something related to this area in graphana. I would recommend looking into how `current` is set when theres no `allValue`. For example if the allValue is `""` i believe `current` is set to `null`. But if there is an `allValue` then it's used